src/wordcab_transcribe/services/asr_service.py

```diff
@@ -300,6 +301,9 @@ def __init__(
                 "You provided URLs for remote diarization server, no local model will"
                 " be used."
             )
+            logger.info(
+                f"remote diarization server timeout is {settings.remote_diarization_request_timeout}"
```

Comment:
`f"Remote diarization server timeout set to {settings.remote_diarization_request_timeout}"`